test/base/test_batch.py

```diff
@@ -323,9 +323,28 @@ def test_utils_to_torch():
     assert batch_torch_float.a.dtype == torch.float32
     assert batch_torch_float.b.c.dtype == torch.float32
     assert batch_torch_float.b.d.dtype == torch.float32
-    array_list = [float('nan'), 1.0]
-    assert to_torch(array_list).dtype == torch.float64
-
+    data_list = [float('nan'), 1]
+    data_list_torch = to_torch(data_list)
+    assert data_list_torch.dtype == torch.float64
+    data_list_2 = [np.zeros((3, 3)), np.zeros((3, 3))]
+    data_list_2_torch = to_torch(data_list_2)
+    assert data_list_2_torch.shape == (2, 3, 3)
+    data_list_3 = [np.zeros((3, 2)), np.zeros((3, 3))]
+    data_list_3_torch = to_torch(data_list_3)
+    assert isinstance(data_list_3_torch, list)
+    assert isinstance(data_list_3_torch[0], torch.Tensor)
+    data_list_4 = [np.zeros((2, 3)), np.zeros((3, 3))]
+    data_list_4_torch = to_torch(data_list_4)
+    assert isinstance(data_list_4_torch, list)
+    assert isinstance(data_list_4_torch[0], torch.Tensor)
+    data_list_5 = [np.zeros(2), np.zeros((3, 3))]
+    data_list_5_torch = to_torch(data_list_5)
+    assert isinstance(data_list_5_torch, list)
+    assert isinstance(data_list_5_torch[0], torch.Tensor)
+    data_array = np.zeros((3, 2, 2))
+    data_tensor = to_torch(data_array[[]])
+    assert isinstance(data_tensor, torch.Tensor)
+    assert data_tensor.shape == (0, 2, 2)
```

Comment:
Also with `np.allclose(to_numpy(data_tensor), data_array)`

Reply:
I mean to test both to_numpy and to_torch that won't fail on this (0, 2, 2) case

Reply:
Let me know if it is fine now.